libs/community/langchain_community/llms/mlrun.py

```diff
@@ -0,0 +1,170 @@
+from collections.abc import AsyncIterator, Iterator
+from typing import Any, List, Optional, Union
+
+import mlrun
+from langchain_core.language_models.base import LanguageModelInput
+from langchain_core.language_models.llms import LLM
+from langchain_core.pydantic_v1 import Field
+from langchain_core.runnables import RunnableConfig
+
+
+class Mlrun(LLM):
+    """A class for interacting with Langchain, while using models served by mlrun."""
+
+    llm: Any = Field(default=None)
+    name: str = Field(default=None)
```

Comment:
why did you choose to rename the arguments from the `__init__`?
```suggestion
    mlrun_function: Union[str, mlrun.runtimes.RemoteRuntime, mlrun.serving.server.GraphServer] = Field(default=None)
    model_name: str = Field(default=None)
```